integration/mediation-tests/tests-other/src/test/resources/testng.xml

```diff
@@ -62,6 +62,13 @@
             <package name="org.wso2.carbon.esb.contenttype.json.*"/>
         </packages>
     </test-->
+
+    <test name="Content-Type-Test" preserve-order="true" verbose="2">
+        <classes>
+            <class name="org.wso2.carbon.esb.contenttype.json.XMLToJsonNilTestCase"/>
```

Comment:
Shall we add to an existing test module if exist. It is useful when we need to disable a set of test cases and also it is bit ugly when a new test is added for each class.

Reply:
The existing test module which is relevant to this has already commented out as below and that's why I've added this as a new module
https://github.com/wso2/product-ei/blob/master/integration/mediation-tests/tests-other/src/test/resources/testng.xml#L58